custom_actions/__init__.py

```diff
@@ -0,0 +1,24 @@
+# This is the standard FastAPI 'hello world' example.
+from fastapi import FastAPI
+import uvicorn
+
+
+app = FastAPI()
+
+async def app(scope, receive, send):
```

Comment:
I do wonder if this file actually needs to be here. Doesn't feel like we need it.

Reply:
I'll remove it